0-bootstrap/modules/jenkins/variables.tf

```diff
@@ -86,20 +112,20 @@ variable "project_prefix" {
 }
 
 variable "activate_apis" {
-  description = "List of APIs to enable in the Cloudbuild project."
+  description = "List of APIs to enable in the CICD project."
   type        = list(string)
 
   default = [
     "serviceusage.googleapis.com",
     "servicenetworking.googleapis.com",
     "compute.googleapis.com",
     "logging.googleapis.com",
-    "bigquery.googleapis.com",
+    "bigquery.googleapis.com", // TODO(caleonardo): confirm if CICD Project needs BQ
```

Comment:
This is required to create the BQ log sinks

Reply:
Great. Thank you.